.github/workflows/build-and-test-typescript.yml

```diff
@@ -0,0 +1,30 @@
+name: Build and Test TypeScript
+on:
+  push:
+    paths:
+      - typescript/**
+  workflow_dispatch:
+
+env:
+  PVAULT_SERVICE_LICENSE: ${{ secrets.PVAULT_SERVICE_LICENSE }}
+
+jobs:
+ test-python:
+   name: Test TypeScript - Getting Started.
+   runs-on: ubuntu-latest
+   steps:
+     - name: Check Out Repo.
+       uses: actions/checkout@v3
+
+     - name: Install Node.js
+       uses: actions/setup-node@v3
+       with:
+         node-version: '18.x'
+         cache: "yarn"
+         cache-dependency-path: typescript/yarn.lock
+
+     - name: Start Vault and run tests.
+       working-directory: typescript
+       run: |
+         yarn
+         yarn test
```

Comment:
./prepare.sh ./run.sh?

Reply:
This is our own CI so I'm OK with using yarn here because I know we have it available.

Reply:
I changed the CI to use the prepare script but I'm still using `npm test` and not `./run.sh` because `run.sh` runs internally `npm start` which expect a Vault to be running while `npm test` use the `@piiano/testcontainers-vault` to start it.